src/core/mod.rs

```diff
@@ -27,6 +27,7 @@ impl Side {
     }
 }
 
+pub const ZERO: Size = Size::new(0, 0);
```

Comment:
This should go inside the `impl Size` block, so that its full path would be `sapling::core::Size::ZERO`.  Also the code looks nicer if all definitions have empty lines on both sides :wink:.